recipe/meta.yaml

```diff
@@ -35,14 +35,15 @@ requirements:
     - xorg-kbproto
     - xorg-libice 1.0.*
     - xorg-libsm 1.2.*
-    - xorg-libx11 1.6.*
+    - xorg-libx11 >=1.6,<2
+    - xorg-libx11
```

Comment:
I would like to add yet another explanatory comment here.

Reply:
I knew I missed something. Thanks for pointing out.